### PR TITLE
Fix spaces link to dev-docs

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #384 (Broken link to spaces dev guide)

* Fixes a broken link
* Improves the description text

### CODE REVIEW

File: modules/ROOT/pages/architecture/architecture.adoc [hunk 1]
```diff
@@ -55,11 +55,11 @@ image::architecture/c4-ocis-system-container.svg[Infinite Scale Container View]
 
 ==== Spaces
 
-A storage space is a logical concept. It organizes a set of resources in a hierarchical tree. It is identified by a `storage space id`, but no ownership is needed. Spaces can have users or groups, a quota, permissions etc. assigned. They may serve different purposes and have different workflows enabled like antivirus scanning. Examples for spaces would be every user’s personal storage space, project or group storage spaces including shares and reshares.
+A storage space is a logical concept. It organizes a set of resources in a hierarchical tree. It is identified by a `storage space id`, but no ownership is needed though one or more users are assigned to manage this space. Spaces can have users or groups, a quota, permissions, an image and a description etc. assigned. They may serve different purposes and have different workflows enabled like antivirus scanning. Examples for spaces would be every user’s personal storage space, project or group or class storage spaces including shares and reshares.
```
Comment (phil-davis) [inline modules/ROOT/pages/architecture/architecture.adoc:58]: should we say "school class" or something?
I am not sure. Technical readers also have an OOP meaning of "class" in their heads!

**Review by phil-davis — APPROVED** [review]

File: modules/ROOT/pages/architecture/architecture.adoc (hunk 1 above)
Comment (mmattel) [reply modules/ROOT/pages/architecture/architecture.adoc:58]: good point, I think "school class" is better.

Comment (mmattel) [inline modules/ROOT/pages/architecture/architecture.adoc:58]:
```suggestion
A storage space is a logical concept. It organizes a set of resources in a hierarchical tree. It is identified by a `storage space id`, but no ownership is needed though one or more users are assigned to manage this space. Spaces can have users or groups, a quota, permissions, an image and a description etc. assigned. They may serve different purposes and have different workflows enabled like antivirus scanning. Examples for spaces would be every user’s personal storage space, project, group or school class storage spaces including shares and reshares.
```

Comment (mmattel) [reply modules/ROOT/pages/architecture/architecture.adoc]: changed

**Pull request merged**